ready_recipes/arm_pyart/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [ `uname` == Darwin ]; then
+    # Remove gfortran so that all fortran Py-ART modules are not built.  This
+    # avoids the need to package the gfortran run time in the conda package.
```

Comment:
I've never had success getting Fortan derived extension modules to work reliably on OS X or Linux.  There is no "default" Fortran compiler on Mac (gfortran and libgfortran on Travis CI are provided by brew so cannot be relied upon to exist on a users machine) and the same glibc "match the lowest version" issue is required when linking with libgfortran.

Py-ART only has a single Fortran extension module which is optional so the package is still fully functional when built on a system without Fortran.  Rather than try to figure out how to get these modules to work I'd prefer to just build Py-ART without this small feature.  

Longer term the plan is to re-write the Fortran module in Cython.  The amount of code to port is not that large, just need to find time to do the work.